src/main/java/com/minecolonies/compatibility/Compatibility.java

```diff
@@ -1,12 +1,14 @@
 package com.minecolonies.compatibility;
 
 import com.minecolonies.compatibility.tinkers.SlimeTreeCheck;
+import com.minecolonies.compatibility.tinkers.TinkersWeaponHelper;
 import com.minecolonies.compatibility.tinkers.ToolBrokenCheck;
 import net.minecraft.block.Block;
 import net.minecraft.block.state.IBlockState;
 import net.minecraft.item.ItemStack;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
+import slimeknights.tconstruct.library.utils.ToolHelper;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'slimeknights.tconstruct.library.utils.ToolHelper'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)